randomScripts/HybridNew/creatJobs_HybridNew.py

```diff
@@ -0,0 +1,43 @@
+import os
+import re
+import sys
+
+def checkFile(fi):
+    if os.path.isfile(fi):
+        if os.path.getsize(fi) > 6500:
+            return True
+    return False
+
+if len(sys.argv) < 3:
+    print("Usage: python script.py <directory_path1> <directory_path2>")
+    sys.exit(1)
+
+directory_path1 = sys.argv[1]
+directory_path2 = sys.argv[2]
+
+iJ = 0
+for f in os.listdir(directory_path1):
+    if not f.startswith("Combined") or "txt" not in f:
+        continue
+    if any(substring in f for substring in ["0Sig", "Scaled", "Closure"]) or "higgsCombine" in f:
```

Comment:
Let's live it as is, but this can be a source of frustration if we forget and we eventually want to do toys for the Scaled cards, no?